newIDE/app/src/EventsSheet/ParameterFields/GenericExpressionField/ExpressionParametersEditorDialog.js

```diff
@@ -59,7 +59,11 @@ export default class ExpressionParametersEditorDialog extends Component<
 
     return (
       <Dialog
-        title={<Trans>Enter the expression parameters</Trans>}
+        title={<>
+          <Trans>Enter the expression parameters</Trans>
+          <br/>
+          <p style={{fontSize: "18px", marginTop: "-3px"}}>{expressionMetadata.getDescription()}</p>
```

Comment:
The dialog title is unfortunately not really meant to support a "subtitle" like this. We would also be risking inconsistencies in the interface if we do things like this, because in the future another dialog could do the same with a different fontSize, or an extra br, or a different marginTop...

In this situation I would simply recommend to put the description in a `<Text>` component, before the `<ExpressionParametersEditor>` :) 
It's simpler, you keep the consistent title styling with the other dialogs, and you benefit from `Text`, a component ready made and ensuring the size of the text will be consistent with the whole application - and if we change it one day, we have a single component to change, instead of 100 Dialog having their own fontSize :)